kpi/tasks.py

```diff
@@ -29,12 +30,12 @@ def export_in_background(export_task_uid):
 
 
 @celery_app.task
-def project_view_export_in_background(
-    export_task_uid: str, username: str
+def export_task_in_background(
+    export_task_uid: str, username: str, export_task_name: str
 ) -> None:
     user = User.objects.get(username=username)
-
-    export_task = ProjectViewExportTask.objects.get(uid=export_task_uid)
+    export_task_class = apps.get_model(f'kpi.{export_task_name}')
```

Comment:
I believe this will break if we have tasks defined outside `kpi/tasks.py.` Better to just pass the fully qualified name of the task model.